pom.xml

```diff
@@ -2843,6 +2893,39 @@
         <tag>HEAD</tag>
     </scm>
 
+    <developers>
```

Comment:
Did you copy this from the geomesa-geoserver bit?  (If so, great!)

Reply:
yep :)